### PR TITLE
Remove app name from run command in template's README

### PR DESCRIPTION
**Pull request by sleepyfran**
Quickly applying the suggestion that Angel left on the other PR, since I can't push stuff directly into master 😃

### CODE REVIEW

**Comment by sleepyfran** [comment]
@JordanMarr, @AngelMunoz can I get a review over here? 😃

**Review by AngelMunoz — APPROVED** [review]
sorry for the delay

**Comment by sleepyfran** [comment]
No worries, thanks @AngelMunoz turns out I need the review of someone with write access 😅 Sorry for the confusion.

So I need @JordanMarr or @JaggerJo approval

**Pull request merged**
